### ISSUE TITLE
Open Source License for this project

### ISSUE THREAD

**Issue by jackkwok** [sunnyxiaohu/R-C3D.pytorch#29]
Thanks for open sourcing your work.  Can you add an open source license to this project?

**Comment by minjikim0202** [comment]
Yes, thank you for the open source code. I was wondering though if there's any update on adding an open source license to this project?
